packages/node_modules/@webex/webex-core/src/lib/services/service-catalogs.js

```diff
@@ -1,6 +1,6 @@
 import {
   SERVICE_CATALOGS,
-  SERVICE_CATALOGS_ENUM_TYPES as ECET
+  SERVICE_CATALOGS_ENUM_TYPES as SCET
```

Comment:
Fixed the constant namespace for this file.

Reply:
Not a big fan of shortening these constants. While it does make the code shorter, it requires a lot more knowledge overhead while reading the code.

Reply:
Addressed this, utilizing the root namespace now.